### PR TITLE
Closes geosolutions-it/austrocontrol-C125#13

### PR DESCRIPTION
**Pull request by kappu72**
## Description
Added annotations import

## Issues
 - Fix  geosolutions-it/austrocontrol-C125#13
 

**Please check if the PR fulfills these requirements**
- [X] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)

 - [ ] Bugfix
 - [X] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)
It's impossible to import annotations

**What is the new behavior?**
It's now possible to import annotations

**Does this PR introduce a breaking change?** (check one with "x", remove the other)

 - [ ] Yes
 - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

### CODE REVIEW

File: web/client/components/mapcontrols/annotations/SelectAnnotationsFile.jsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        diableOvveride: React.PropTypes.bool
+    };
+
+    static contextTypes = {
+        messages: PropTypes.object
+    };
+
+    static defaultProps = {
+        show: false,
+        text: "Drop your file here or click to select the Annotation File to import. (supported files: JSON)",
```
Comment (MV88) [inline web/client/components/mapcontrols/annotations/SelectAnnotationsFile.jsx]: is this not localized?

File: web/client/utils/__tests__/FileUtils-test.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+var expect = require('expect');
+var FileUtils = require('../FileUtils');
+
+describe('FilterUtils', () => {
+    it('Test read local json file', (done) => {
+        const jsonFile = new File(["[]"], "file.json", {
+          type: "text/plain"
```
Comment (MV88) [inline web/client/utils/__tests__/FileUtils-test.js]: why the type is text/plain?

File: web/client/components/mapcontrols/annotations/SelectAnnotationsFile.jsx [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    render() {
+        return (
+            <ResizableModal title={this.props.title} bodyClassName="ms-flex" show={this.props.show} showClose onClose={this.props.onClose} size="sm">
+                    <div className="" style={{flexDirection: "column"}}>
+                    {this.state.error && this.renderError()}
+                        <Dropzone accept="application/json" onDropRejected={this.checkfile} rejectClassName="ms-alert ms-alert-center text-center alert-danger" className="ms-alert ms-alert-center text-center" onDrop={this.checkfile}>
+                            <div className="alert alert-info" role="alert" style={{margin: 15, flex: 1}}>{this.state.loading && <Spinner spinnerName="circle" overrideSpinnerClassName="spinner"/>}{this.props.text}</div>
+                        </Dropzone>
+                        <div style={{margin: 15, flex: 1}}>
+                        <Checkbox disabled={this.props.diableOvveride} checked={this.state.override} onChange={() => this.setState(() => ({override: !this.state.override}))}>
```
Comment (MV88) [inline web/client/components/mapcontrols/annotations/SelectAnnotationsFile.jsx]: typo diableOvveride

File: web/client/components/mapcontrols/annotations/SelectAnnotationsFile.jsx [hunk 4]
```diff
@@ -0,0 +1,120 @@
+/*
```
Comment (MV88) [inline web/client/components/mapcontrols/annotations/SelectAnnotationsFile.jsx]: There are some strange style effect if you try to drop the file
initially it appears a div with red background 
image

then for a quick moment the dropzone moves down 
image

can you check this behaviour?

File: web/client/components/mapcontrols/annotations/SelectAnnotationsFile.jsx (hunk 3 above)
Comment (MV88) [inline web/client/components/mapcontrols/annotations/SelectAnnotationsFile.jsx]: if i import twice the same annotation when i go to delete it it removes all the annotations with that name.
is this ok for you @mbarto?

**Comment by MV88** [comment]
I've seen that you have selected Yes for braking change.
is it a mistake? if so could you change it or motivate otherwise. 
Thank you

File: web/client/components/mapcontrols/annotations/SelectAnnotationsFile.jsx (hunk 1 above)
Comment (kappu72) [reply web/client/components/mapcontrols/annotations/SelectAnnotationsFile.jsx]: Yes it is
The message is passed from the container componet

File: web/client/components/mapcontrols/annotations/SelectAnnotationsFile.jsx (hunk 4 above)
Comment (kappu72) [reply web/client/components/mapcontrols/annotations/SelectAnnotationsFile.jsx]: It's a dropzone effect in windows that doesn't have the correct file type. We can remove the dropzone check if you prefer.

File: web/client/components/mapcontrols/annotations/SelectAnnotationsFile.jsx (hunk 3 above)
Comment (kappu72) [reply web/client/components/mapcontrols/annotations/SelectAnnotationsFile.jsx]: This is why you import twice the same annotation.

File: web/client/utils/__tests__/FileUtils-test.js (hunk 2 above)
Comment (kappu72) [reply web/client/utils/__tests__/FileUtils-test.js]: It's ignored but I'll change

**Pull request merged**